pkg/apis/configuration/validation/validation.go

```diff
@@ -156,6 +159,40 @@ func validateUpstreams(upstreams []v1alpha1.Upstream, fieldPath *field.Path, isP
 	return allErrs, upstreamNames
 }
 
+var validNextUpstreamParams = map[string]bool{
+	"error":          true,
+	"timeout":        true,
+	"invalid_header": true,
+	"http_500":       true,
+	"http_502":       true,
+	"http_503":       true,
+	"http_504":       true,
+	"http_403":       true,
+	"http_404":       true,
+	"http_429":       true,
+	"non_idempotent": true,
+	"off":            true,
+	"":               true,
```

Comment:
this is the recent change. better to handle that case in the `validateNextUpstream`. similar to https://github.com/nginxinc/kubernetes-ingress/blob/d9f88740990e27bf12fe44e0fbf9a1e20cd64e59/pkg/apis/configuration/validation/validation.go#L76
Better to add this case to the `TestValidateNextUpstream`